config/contrib.json

```diff
@@ -6,7 +6,7 @@
     "versions": {
       "storage": {
         "git": {
-          "prefixMessageToSnapshotId": "This version was recorded after filtering snapshot with Mongo ID ",
+          "snapshotIdentiferTemplate": "mongo://contrib/open-terms-archive/snapshots/%SNAPSHOT_ID",
```

Comment:
Should that be `mongo://open-terms-archive/contrib/snapshots/%SNAPSHOT_ID` instead of `mongo://contrib/open-terms-archive/snapshots/%SNAPSHOT_ID`?

Reply:
Nope. `mongo://host/database/collection/index`